_data/compatibility/abra.yaml

```diff
@@ -0,0 +1,62 @@
+---
+name: Abra
+internal_url: /en/compatibility/abra
+logo: /img/logos/third-party/abra.png
+description: >
+  TODO
```

Comment:
Since the wallet description isn't used anywhere, perhaps it should be dropped from the schema and all files.